platform/jewel/ui/src/main/kotlin/org/jetbrains/jewel/ui/component/Tooltip.kt

```diff
@@ -60,10 +72,32 @@ public fun Tooltip(
     enabled: Boolean = true,
     style: TooltipStyle = JewelTheme.tooltipStyle,
     tooltipPlacement: TooltipPlacement = style.metrics.placement,
+    neverHide: Boolean = true,
+    autoHideDelay: Duration = style.metrics.regularDisappearDelay,
```

Comment:
IMO it would make more sense to have an `AutoHideBehavior` enum here, which has three values: `Never` (default), `Normal`, `Long`.